geode-core/src/main/java/org/apache/geode/cache/Region.java

```diff
@@ -1362,49 +1362,77 @@ Object selectValue(String queryPredicate) throws FunctionDomainException, TypeMi
    * <code>Scope.LOCAL</code>.
    *
    * If an exception is thrown due to this call, it can imply that there may have been a partial
-   * update performed on the region. Use putAll from within a transaction to to obtain an atomic
-   * update.
+   * update performed on the region.
    * <p>
    *
    * @see java.util.Map#putAll(Map map)
-   * @param map the key/value pairs to put in this region.
+   * @param map the key/value pairs to put in this region
+   * @throws ServerOperationException if called from a client, and the server throws an exception
+   *         such as CacheWriterException, PartitionedRegionStorageException or LowMemoryException.
+   *         These server exceptions become the ServerOperationException cause
+   * @throws ServerConnectivityException if called from a client, and the server throws
+   *         CancelException. CancelException will be the ServerConnectivityException cause
    * @throws NullPointerException if key is null or if value is null (use invalidate instead), or if
    *         the key or value do not meet serializability requirements
    * @throws ClassCastException if key does not satisfy the keyConstraint
-   * @throws org.apache.geode.distributed.LeaseExpiredException if lease expired on distributed lock
-   *         for Scope.GLOBAL
-   * @throws TimeoutException if timed out getting distributed lock for Scope.GLOBAL
+   * @throws org.apache.geode.distributed.LeaseExpiredException if lease expired on distributed
+   *         lock for regions with Scope.GLOBAL
```

Comment:
Please slightly reword LeaseExpiredException (in BOTH putall signatures):
if the lease expired on a distributed lock, for regions with Scope.GLOBAL

Reply:
fixed